### ISSUE TITLE
Why the snap isn't simply called retro-virtual-machine or rvm?

### ISSUE THREAD

**Issue by Recydywa** [mmartinortiz/retro-virtual-machine-snap#4]
Why you decided to add your GitHub account name at the beggining?

This is extremly counterintuitive. I tried installing snap from terminal. First I tried `retro-virtual-machine` but tab completion didn't want to complete `retro- `so I typed everything manually but the snap didn't exist.

Than I thought maybe it is RVM, but still no success.

I had to visit the Snap Store website and then I had the very not good name of the app.

You may say that this isn't a problem if one installs from App Center, but I prefer the terminal. For one reason I can install or remove multiple snaps, typing my root password just once, and in the App Center it is requierd for evry single operation.

Also the command to launch the RVM snap is unnecessary complicated and again it contains your GitHub user name at the beginning. This is the first snap that I came across that uses that naming convention.

**Comment by mmartinortiz** [comment]
That is a good question and I'm afraid I do not have a good answer.

This was my second snap, the first and only one I published. Back then I followed a tutorial from Canonical. Maybe the tutorial used the username as part of the snap, or maybe I thought that, since this is not an official build from the RVM author, was not appropriate to take the name.

I fully agree that the snap name is completely not intuitive from the command line.

After a quick look to the consequences of renaming the snap, I found [this question](https://forum.snapcraft.io/t/rename-a-snap/31143) without answer, this [other old answer](https://forum.snapcraft.io/t/rename-existing-snap/26212) indicating that renamed was not supported back them, and [this other question](https://forum.snapcraft.io/t/snap-rename-on-upstream-project-changes/10403) which also lacks an answer.

I'm afraid that we have to live with this name :man_shrugging:
